cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageGdpr.kt

```diff
@@ -231,9 +233,17 @@ private class DataStorageGdprImpl(context: Context) : DataStorageGdpr {
     }
 
     override fun clearGdprConsent() {
+
+        val spEditor = preference.edit()
+        preference
+            .all
+            .filter { it.key.startsWith("IABTCF_") }
```

Comment:
Here you could use the constant you defined above `IABTCF_KEY_PREFIX`.

Reply:
true